docs/TuxMutableClass.md

```diff
@@ -0,0 +1,142 @@
+
+The mutable class in `Tux` is an extension of the owneeClass [***link here***](tuxedojs.org/docs/TuxReact/TuxOwneeClass).
+
+### <a id="Requiring-TuxMutableClass"></a>1) Requiring Tux Mutable Class [#](#Requiring Tux Mutable Class)
+The Tux Mutable Class method `createMutableClass` is exposed by requiring the main Tux module, which can be required via:
+
+```javascript
+  var React = require('tux');
+```
+
+**Note that at this time for Tux to be compiled by the JSX transformer, the Tux module must be assigned to the variable name `React`**
+
+### <a id="creating-mutable-class-components"></a>2) Creating Mutable Class Components [#](#creating-mutable-class-components)
+The Tux extended `React` object allows users to create mutable class components via invoking [React.createMutableClass](#createMutableClass). Pass in an object to the `createMutableClass` method and it will return a new object that has been extended with `componentWillMount` and `shouldComponentUpdate` React life cycle methods. The `createMutableClass` method is just like the `React.createClass` method, except it has extended `mutableClass` functionality.
+
+Let's take a look at an example: [#](#create-mutable-class-example)
+<a id="create-mutable-class-example"></a>
+
+```javascript
+  var Message = React.createMutableClass({
+    // the traits we want to watch to see if a re-render is required
+    mutableTraits: {
+      props: ['text', 'timestamp'],
+      state: 'editing'
+    },
+
+    render: function () {
+      return (
+        <div>
+          <h1>{this.props.message.text}</h1>
+        </div>
+      );
+    }
+  });
+```
+
+Below is the api documentation for this method.
+
+### <a id="React-createMutableClass"></a>React.createMutableClass [#](#React-createMutableClass)
+
+```javascript
+  var Message = React.createMutableClass(mutableClassProps);
+```
+
+#### <a id="createMutableClass-mutableClassProps"></a>Parameter - `mutableClassProps` - type: OBJECT - required [#](#createMutableClass-mutableClassProps)
+Methods and properties to be extended onto the `mutableClass` object returned. Expected keys:
+
+##### <a id="mutableClassProps-mutableTraits"></a>Property - `mutableClassProps.mutableTraits` - type: OBJECT - optional [#](#mutableClassProps-mutableTraits)
+The traits that will be registered onto the mutable class.
+
+**NOTE: If the `mutableTraits` property is not included in the `mutableClassProps` object, instead of applying the TuxMutableRenderMixin to the return object, the PureRenderMixin will be applied instead. The PureRenderMixin implements `shouldComponentUpdate`, but it only does a shallow compare compared to the deep search that is done in the TuxMutableRenderMixin.**
```

Comment:
Add a link to the PureRenderMixin docs in React.  Change to ...but it only does a shallow compare of the top-most keys in props and state.